### PR TITLE
Logout event in testRunner

### PR DESCRIPTION
**Pull request by zShurick**
https://oat-sa.atlassian.net/browse/TAO-2101
depends on https://github.com/oat-sa/tao-core/pull/721

### CODE REVIEW

File: views/js/controller/runtime/service/deliveryExecution.js [hunk 1]
```diff
@@ -23,8 +23,9 @@ define([
     'helpers',
     'taoDelivery/controller/runtime/service/fullScreen',
     'layout/loading-bar',
-    'ui/dialog/alert'
-], function(_, $, helpers, fullScreen, loadingBar, dialogAlert){
+    'ui/dialog/alert',
+    'layout/logout-event'
+], function(_, $, helpers, fullScreen, loadingBar, dialogAlert, logoutEvent){
```
Comment (krampstudio) [inline views/js/controller/runtime/service/deliveryExecution.js]: Double check please if `helpers` is still in use. If not, please remove the dependency.

**Comment by zShurick** [comment]
@krampstudio fixed

**Comment by bugalot** [comment]
Hi @zShurick !
This is working very well!

However, when being disconnected (session timeout) while being taking a Test as a Test Taker, here is what happens:

1. The session goes timeout
2. I click the "Next Button".
3. First, the "An error occurs, please contact your administrator" pop-up appears. This is because the item runner receives a HTTP forbidden response from the server. I click "Ok".
4. The new pop-up with text "You have been disconnected, please login again" appears. That's what we want. I click "Ok" and I'm correctly redirected to the Login screen.

What we expect is that step 3. does not appear in case of session timeout (403). However, if a real error (500) occurs, it should appear.

Does that make sense to you?

**Comment by zShurick** [comment]
Hi @bugalot 
I can't reproduce the alert. Even I'm got 500

**Comment by zShurick** [comment]
@bugalot 
I found, thanks

**Comment by zShurick** [comment]
@bugalot 
fixed
https://github.com/oat-sa/extension-tao-testqti/pull/310

**Comment by zShurick** [comment]
@bugalot And that PR to, until the evening, please

**Comment by llecaque** [comment]
please increase version number

**Comment by zShurick** [comment]
@llecaque version of the extension tao-delivery? or version of the tao-core in the manifest?

**Pull request merged**